src/test/java/com/wikia/webdriver/common/core/api/ApiCall.java

```diff
@@ -59,9 +62,11 @@ public void call() {
         httpPost.setEntity(new UrlEncodedFormEntity(getParams(), StandardCharsets.UTF_8));
       }
 
-      httpClient.execute(httpPost);
+      CloseableHttpResponse resp = httpClient.execute(httpPost);
 
-      PageObjectLogging.logInfo("CONTENT PUSH: ", "Content posted to: " + getURL());
+      PageObjectLogging.logInfo("CONTENT PUSH: ", "Header: " + httpPost.getHeaders("X-Wikia-AccessToken")[0]);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "CONTENT PUSH: " 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)